zerver/models.py

```diff
@@ -966,6 +966,19 @@ def __str__(self) -> str:
         return f"<RealmPlayground({self.realm.string_id}): {self.pygments_language} {self.name}>"
 
 
+def get_realm_playgrounds(realm: Realm) -> List[Dict[str, str]]:
+    playgrounds: List[Dict[str, str]] = []
+    for playground in RealmPlayground.objects.filter(realm=realm).all():
+        playgrounds.append(
+            dict(
+                name=playground.name,
+                pygments_language=playground.pygments_language,
+                url_prefix=playground.url_prefix,
+            )
+        )
+    return playgrounds
```

Comment:
I'd squash this into the first commit using it -- we don't like adding a function in a commit that doesn't have tests for it, since it's incomplete in that sense.